tabular/src/autogluon/tabular/models/text_prediction/text_prediction_v1_model.py

```diff
@@ -62,8 +65,21 @@ def __init__(self, **kwargs):
         """
         super().__init__(**kwargs)
         self._label_column_name = None
-        self._numeric_columns = None
-        self._cat_columns = None
+        self._feature_generator = None
+
+    def _preprocess(self, X, fit=False, **kwargs):
+        from ...features.feature_metadata import R_CATEGORY
+        if fit:
+            from ...features.generators import BulkFeatureGenerator, CategoryFeatureGenerator, IdentityFeatureGenerator
```

Comment:
One minor question. I feel that these feature generators are super useful. Will we later move that to core?

Reply:
The plan is to actually move them to a new module called `autogluon.features`, which will only depend on `autogluon.core`. This is the next major effort planned after API refactor. The hope is for `autogluon.features` to be in v0.1 release.